### ISSUE TITLE
Visual Studio 2015: pressing Ctrl+S while the check is running triggers the "Save As" dialog

### ISSUE THREAD

**Issue by VioletGiraffe** [VioletGiraffe/cppcheck-vs-addin#134]
Looks like VS is thinking the file is busy or inaccessible and offers to save it under a different name instead. Odd.

**Comment by zchrissirhcz** [comment]
Hi @VioletGiraffe, I met this problem in my VS2017. I'm used to press Ctrl+S to save my current edited file, and the always pop-up "save cppcheck result" window is not what I expected. Is there any setting that we can do to avoid that?

**Comment by VioletGiraffe** [comment]
I have the same habit of pressing Ctrl+S very often, and I encountered the same problem a long time and haven't been able to figure it out. I thought the source file is locked while cppcheck is running, so I added a short delay before cppcheck is started hoping that it will be enough for Visual Studio to save the file, but it doesn't always help (although it seems to have improved things, the dialog pops up less often).

If in your case it was simply a request to locate cppcheck.exe, then we could do with a better UI for that. I'll do something about it.
